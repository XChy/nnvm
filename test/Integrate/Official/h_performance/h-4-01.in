version https://git-lfs.github.com/spec/v1
oid sha256:5fd843a5a31d06c812ad6d05489a49b0ecdfc351ec0167275835891eade3d778
size 73
