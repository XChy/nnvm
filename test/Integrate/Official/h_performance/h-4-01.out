1
829569550
190386254
0
